eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala

```diff
@@ -124,11 +124,11 @@ class EclairImplSpec extends TestKitBaseClass with FixtureAnyFunSuiteLike with I
 
     // with assisted routes
     val externalId1 = "030bb6a5e0c6b203c7e2180fb78c7ba4bdce46126761d8201b91ddac089cdecc87"
-    val hints = List(List(ExtraHop(Bob.nodeParams.nodeId, ShortChannelId("569178x2331x1"), feeBase = 10 msat, feeProportionalMillionths = 1, cltvExpiryDelta = CltvExpiryDelta(12))))
+    val hints = List(List(ExtraHop(Bob.nodeParams.nodeId, ShortChannelId.fromCoordinates("569178x2331x1").get, feeBase = 10 msat, feeProportionalMillionths = 1, cltvExpiryDelta = CltvExpiryDelta(12))))
```

Comment:
In theory we should use Scalatest's [TryValues](https://www.scalatest.org/scaladoc/3.0.6/org/scalatest/TryValues.html) here.

Reply:
Done in https://github.com/ACINQ/eclair/pull/2320/commits/74e19803a34225e3b36948af2cc8cb09958f5783